modules/op-release-notes-1-6.adoc

```diff
@@ -0,0 +1,194 @@
+
+* This update adds an `ignore-running` flag to the `tkn pipelinerun delete` and `tkn taskrun delete` commands. (link:https://github.com/tektoncd/cli/pull/1445[#1445])
+
+* This update adds a `create` subcommand to the `tkn task` and `tkn clustertask` commands. (link:https://github.com/tektoncd/cli/pull/1359[#1359])
+
+=== CLI 0.20.0 release
+
+* With this update, if you use the `tkn pipelinerun delete --all` command, you can preserve runs that are younger than a specified number of minutes by using the new `--keep-since <minutes>` option. For example, to keep pipeline runs that are less than five minutes old, you enter `$ tkn pipelinerun delete -all --keep-since 5`. (link:https://github.com/tektoncd/cli/pull/1392[#1392])
+
+* With this update, when you use the `tkn pipelinerun delete --all` command, you can use the new `--label <string>` option to filter the pipeline runs by label. Optionally, you can use the `--label` option with `=` and `==` as equality operators, or `!=` as an inequality operator. For example, the `tkn pipelinerun delete --all --label asdf` and  `tkn pipelinerun delete --all --label==asdf` commands both delete all the pipeline runs that have the `asdf` label. (link:https://github.com/tektoncd/cli/pull/1402[#1402])
+
+* With this update, you can fetch the version of installed Tekton components from the `ConfigMap` or, if the `ConfigMap` is not present, from the deployment controller. (link:https://github.com/tektoncd/cli/pull/1393[#1393])
+
+=== Tekton Triggers v0.16.0
+
+* With this update, triggers support the `configmap feature-flags` and `configmap config-defaults` commands to configure feature flags and to set default values, and adds `serviceAccountName` to the defaults. (link:https://github.com/tektoncd/cli/pull/1110[#1110], link:https://github.com/tektoncd/cli/pull/1182[#1182])
+
+* This update adds a new metric, `eventlistener_event_count`, that you can use to count events received by the `EventListener` sink. (link:https://github.com/tektoncd/triggers/pull/1160[#1160])
+
+=== Tekton Triggers v0.15.0
+
+* This update adds `v1beta1` Go API types. (link:https://github.com/tektoncd/triggers/pull/1103[#1103])
++
+With this update, Triggers now supports a new `apiVersion` value, `v1beta1`. The `v1alpha1` version will follow the beta deprecation policy: `v1alpha1` features will be deprecated over a 9-month period.
+
+[id="configuring-node-selection-1-6_{context}"]
+=== Configuring Node Selection
+// TBD - Where does this release note belong?
+
+* The current release enables you to configure node selection by specifying the `nodeSelector` and `toleration` values in the `TektonConfig` custom resource (CR) YAML. The Operator adds these values to all deployments it creates.
```

Comment:
```suggestion
* The current release enables you to configure node selection by specifying the `nodeSelector` and `toleration` values in the `TektonConfig` custom resource (CR). The Operator adds these values to all deployments it creates.
```
probably don't need YAML here, particularly if this can be done using `tkn` or ODC as well, without YAML?

Reply:
I believe this we can done by YAML i think
/cc @sm43

Reply:
Thank you, both. I'm not sure what to make of this feedback - I'll go ahead and remove `YAML`. Let me know if further changes are needed.